scripts/apps/search/components/ItemList.tsx

```diff
@@ -102,7 +102,7 @@ export class ItemList extends React.Component<any, IState> {
         this.setState({narrow: setNarrow});
     }
 
-    select(item, event) {
+    select(item, event, {preview}: {preview?: boolean} = {preview: null}) {
```

Comment:
Would it work removing `CHECKBOX_PARENT_CLASS` so we only have one way of doing the same thing?

Reply:
actually I don't think it should call that select at all when selecting multiple items